javascript/ql/lib/semmle/javascript/internal/flow_summaries/TypedArrays.qll

```diff
@@ -36,3 +36,32 @@ class BufferTypedArray extends DataFlow::AdditionalFlowStep {
     )
   }
 }
+
+class SetLike extends SummarizedCallable {
```

Comment:
The `Like` suffix is generally used when the class models multiple functions that behave similarly, but don't have the same name. I'd just call it `TypedArraySet`